src/main/java/lotto/model/LottoGame.java

```diff
@@ -1,19 +1,44 @@
 package lotto.model;
 
-import lotto.model.vo.PurchaseCount;
-
+import java.util.Arrays;
 import java.util.List;
 import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 import java.util.stream.Stream;
 
 public class LottoGame {
 
+    private static final String SPLIT_REGEX = ",";
+    private static final int ZERO = 0;
+
     private final List<LottoGroup> lottoGroups;
 
-    public LottoGame(final PurchaseCount purchaseCount, final LottoMachine lottoMachine) {
-        this.lottoGroups = Stream.generate(lottoMachine::generate)
-                .limit(purchaseCount.getCount())
-                .collect(Collectors.toUnmodifiableList());
+    public LottoGame(final PurchaseCounts purchaseCounts, final List<String> manualLottoNumbersInput, final LottoMachine lottoMachine) {
+        if (purchaseCounts.getManualPurchaseCount() == 0) {
+            this.lottoGroups = makeRandomLottoGroups(purchaseCounts, lottoMachine)
+                    .collect(Collectors.toUnmodifiableList());
+            return;
+        }
+        this.lottoGroups = Stream.concat(
+                makeManualLottoGroups(manualLottoNumbersInput, lottoMachine),
+                makeRandomLottoGroups(purchaseCounts, lottoMachine)
+        ).collect(Collectors.toUnmodifiableList());
+
+    }
+
+    private static Stream<LottoGroup> makeManualLottoGroups(final List<String> manualLottoNumbersInput, final LottoMachine lottoMachine) {
+        return manualLottoNumbersInput.stream()
+                .map(input -> input.split(SPLIT_REGEX))
+                .map(numbers -> Arrays.stream(numbers)
+                        .map(String::trim)
+                        .map(Integer::parseInt)
+                        .collect(Collectors.toUnmodifiableList())
+                ).map(lottoMachine::manualGenerate);
```

Comment:
map 을 활용하면 좋긴하지만 객체지향적으로 보았을 때 여러 책임이 존재하는 것 처럼 보입니다~
메서드로 분리를 해보면 좋을 것 같아요 :)

Reply:
```suggestion
        return manualLottoNumbersInput.stream()
                .map(input -> input.split(SPLIT_REGEX))
                .map(convertToNumbers())
                .map(lottoMachine::manualGenerate);
    }

    private static Function<String[], List<Integer>> convertToNumbers() {
        return numbers -> Arrays.stream(numbers)
                .map(String::trim)
                .map(Integer::parseInt)
                .collect(Collectors.toUnmodifiableList());
    }
```
convertToNumbers() 라는 메서드를 만들어 분리해 봤습니다!